modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -30,8 +30,7 @@ NOTE: Serverless is currently in a limited availability (LA) release with xref:d
 === Dedicated Cloud
 
 With Dedicated clusters, you host your data in Redpanda's VPC, and Redpanda handles provisioning, operations, and maintenance. Dedicated clusters are single-tenant deployments that support private networking (that is, VPC peering to talk over private IPs) for better data isolation.
-
-Available on AWS or GCP. 
+When you create a Dedicated cluster, you select a xref:deploy:deployment-option/cloud/dedicated/dedicated-tiers.adoc[throughput tier and region] on either AWS or GCP. 
```

Comment:
Why do I need to select a throughput tier. This should be covered somewhere and linked in this paragraph.